src/sentry/api/endpoints/project_backfill_similar_issues_embeddings_records.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.auth.superuser import is_active_superuser
+from sentry.tasks.backfill_seer_grouping_records import backfill_seer_grouping_records
+
+region_silo_endpoint
```

Comment:
```suggestion
```

Reply:
I'm not sure why this keeps re-appearing - it doesn't show up in my code editor locally either